src/Type/SchemaConfig.php

```diff
@@ -24,8 +24,10 @@
  *
  *     $schema = new Schema($config);
  *
- * @phpstan-type TypeLoader callable(string $typeName): ((Type&NamedType)|null)
- * @phpstan-type Types array<Type&NamedType>|(callable(): array<Type&NamedType>)
+ * @see Type, NamedType
```

Comment:
I do this to prevent IDE marking those imports as unused to prevent accidental removal and allow using auto format with import optimization